controller_manager/src/controller_manager.cpp

```diff
@@ -163,6 +164,8 @@ ControllerManager::ControllerManager(
 
   init_resource_manager(robot_description);
 
+  diagnostics_updater_.setHardwareID("ros2_control");
+  diagnostics_updater_.add("Controllers Health", this, &ControllerManager::CreateActiveDiagnostic);
```

Comment:
Also, the name of the diagnostic would be more exact. do you agree?

```suggestion
  diagnostics_updater_.add("Controllers Activity", this, &ControllerManager::controller_activity_diagnostic_callback);
```